backend/Cargo.lock

```diff
@@ -1128,7 +1128,7 @@ checksum = "3354b9ac3fae1ff6755cb6db53683adb661634f67557942dea4facebec0fee4b"
 [[package]]
 name = "utils"
 version = "0.1.0"
-source = "git+https://github.com/a-b-street/utils#10d8bf23282fb601ffc3eeb78b5ea8b9b1acd496"
+source = "git+https://github.com/a-b-street/utils?branch=mkirk%2Fupdate-geo#0c4274485d6295c2e0f3c89d74282400e27449d3"
```

Comment:
I'd be fine merging https://github.com/a-b-street/utils/pull/3, unless there's a reason to wait. If you mark it ready for review, I could do that now, if you like